README.md

```diff
@@ -16,6 +16,19 @@ and external templates including:
 
 Download the extension from [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=Angular.ng-template).
 
+## Versioning
+
+The langauge service extension uses the `@angular/language-service` package for its backend. This
+package is loaded either from the version bundled with the extension, or from the current workspace
+project the extension is running on. Due to the behavior of TypeScript below version 3.8,
+incompatible versions of the language service may sometimes be loaded. If you are using a version of
+TypeScript below 3.8, we suggest either
+
+- Not installing `@angular/language-service` in your project (recommended; will fallback on the
+    version bundled with the extension)
+- Installing and keeping updates for the latest version of `@angular/language-service`
+
+For further information, please see #594.
```

Comment:
Could you please provide a link to #594? This doc gets bundled with the extension and GitHub automatic linking does not work when the document is shown in the editor.